prediction-applier/src/main/java/org/alfresco/hxi_connector/prediction_applier/hxinsight/PredictionMapper.java

```diff
@@ -0,0 +1,45 @@
+/*-
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2023 - 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.hxi_connector.prediction_applier.hxinsight;
+
+import java.util.List;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import org.alfresco.hxi_connector.common.model.prediction.Prediction;
+import org.alfresco.hxi_connector.common.model.repository.Node;
+
+@Component
+@RequiredArgsConstructor
+public class PredictionMapper
+{
+
+    public Node map(Prediction prediction)
```

Comment:
I thought there would be three conversion steps - something like:

1. HxI prediction batch deserialisation object (from REST API) -> Internal prediction batch object
2. Internal prediction batch object -> Internal node prediction task object (gets put onto AMQ)
3. Internal node prediction task object (retrieved from AMQ) -> Alfresco node update request (sent to REST API)

I'm not sure which of the two this step is, or whether this is actually a stage in the middle of step 3?

Reply:
This should correspond to 3. stage - this is mapping of an object received from AMQ and then passed to repo's `/nodes` API. I didn't want to introduce this mapping at all - but I came up with an idea of adding also #281 so I needed some MVP of this prediction flow, and for that I needed some really basic mapping